gradle.properties

```diff
@@ -2,5 +2,6 @@ TILEDB_GIT_REPOSITORY=https://github.com/TileDB-Inc/TileDB
 TILEDB_GIT_TAG=2.0.8
 TILEDB_VERBOSE=OFF
 TILEDB_S3=ON
+TILEDB_AZURE=ON
```

Comment:
Unfortunately Azure and GCS doesn't build on windows currently. Its on our list to fix the builds for but for now it might make more sense to disable azure by default and then enable it for OSX/Linux in CI.

Reply:
👍 

Will do that in the next follow up commits.